lux/_config/config.py

```diff
@@ -137,13 +139,37 @@ def sort(self, flag: Union[str]):
         """
         flag = flag.lower()
         if isinstance(flag, str) and flag in ["none", "ascending", "descending"]:
-            self._sort = flag
+            if flag == "none":
+                self._sort = None
+            elif flag == "ascending":
+                self._sort = False
+            else:
+                self._sort = True
         else:
             warnings.warn(
                 "Parameter to lux.config.sort must be one of the following: 'none', 'ascending', or 'descending'.",
                 stacklevel=2,
             )
 
+    @property
+    def ordering(self):
+        return self._ordering
+
+    @ordering.setter
+    def ordering(self, value):
```

Comment:
Can we rename all `ordering` as `ranking`? We should avoid the word sorter in places where it might be unclear what it is referring it.